client/lib/plans/features-list.js

```diff
@@ -530,7 +530,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_WORDADS_INSTANT ]: {
 		getSlug: () => constants.FEATURE_WORDADS_INSTANT,
-		getTitle: () => i18n.translate( 'Site Monetization' ),
+		getTitle: () => i18n.translate( 'Site monetization' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.